src/vm/CMakeLists.txt

```diff
@@ -245,6 +245,7 @@ set(VM_SOURCES_WKS
     typeparse.cpp
     verifier.cpp
     weakreferencenative.cpp
+    nativeeventsource.cpp
```

Comment:
The files in this list are in alphabetical order. Could you please add nativeeventsource.cpp to the appropriate spot?